### PR TITLE
Load cert default way with App Service

### PR DESCRIPTION
**Pull request by rekhaswaminathan**

### CODE REVIEW

File: src/Diagnostics.DataProviders/KeyVaultCertLoader/KeyVaultCertLoaderBase.cs [hunk 1]
```diff
@@ -1,32 +1,42 @@
...
     {
-        protected abstract string KeyVault { get; set; }
-
-        protected abstract string CertificateName { get; set; }
-
+        protected abstract string Thumbprint { get; set; }
         public X509Certificate2 Cert { get; private set; }
 
-        public async void LoadCertFromKeyVault()
+        public async void LoadCertFromAppService()
```
Comment (dagrooms52) [inline src/Diagnostics.DataProviders/KeyVaultCertLoader/KeyVaultCertLoaderBase.cs]: Using `async void` means callers can't `await` this to get exception details. There is some code outside of `try/catch` in the method body, so this has a possibility of losing an exception or crashing the service.

Comment (rekhaswaminathan) [reply src/Diagnostics.DataProviders/KeyVaultCertLoader/KeyVaultCertLoaderBase.cs]: @dagrooms52 `async` was previously there when we were loading from Key Vault. I missed removing that when I made the change to load from App Service. Should be updated now

File: src/Diagnostics.DataProviders/KeyVaultCertLoader/CertLoaderBase.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                // Get the first cert with the thumbprint
+                if (certCollection.Count > 0)
+                {
+                    Cert = certCollection[0];
+                    DiagnosticsETWProvider.Instance.LogRuntimeHostMessage($"Successfully loaded Cert with thumbprint {Thumbprint}");
+                }
+            }
+            catch (Exception ex)
+            {
+                DiagnosticsETWProvider.Instance.LogRuntimeHostMessage($"Error: {ex.Message} occurred while trying to load cert {Thumbprint} ");
```
Comment (ShekharGupta1988) [inline src/Diagnostics.DataProviders/KeyVaultCertLoader/CertLoaderBase.cs]: Lets include the full stack trace as well in this message.

Comment (rekhaswaminathan) [reply src/Diagnostics.DataProviders/KeyVaultCertLoader/CertLoaderBase.cs]: Sure

**Pull request merged**